environment/src/test/java/com/sequenceiq/environment/environment/encryption/EnvironmentEncryptionServiceTest.java

```diff
@@ -0,0 +1,139 @@
+                .withLocationDto(LocationDto.builder().withName("DummyRegion").build())
+                .withParameters(ParametersDto.builder()
+                        .withAzureParameters(AzureParametersDto.builder()
+                                .withEncryptionParameters(AzureResourceEncryptionParametersDto.builder()
+                                        .withEncryptionKeyUrl("dummyKeyUrl").build())
+                                .withResourceGroup(AzureResourceGroupDto.builder()
+                                        .withResourceGroupUsagePattern(ResourceGroupUsagePattern.USE_SINGLE)
+                                        .withName("dummyRG").build())
+                                .build())
+                        .build())
+                .withTags(new EnvironmentTags(new HashMap<>(), new HashMap<>()))
+                .build();
+        DiskEncryptionSetCreationRequest createdDes = underTest.createEncryptionResourcesCreationRequest(environmentDto);
+
+        assertEquals(createdDes.getEncryptionKeyUrl(), "dummyKeyUrl");
+        assertEquals(createdDes.getEnvironmentId(), 1L);
+        assertEquals(createdDes.getRegion(), Region.region("DummyRegion"));
+        assertEquals(createdDes.getResourceGroup(), "dummyRG");
+        assertEquals(createdDes.isSingleResourceGroup(), true);
+    }
+
+    @Test
+    void testCreateEncryptionResourcesShouldCallCreateDiskEncryptionSetWhenCloudPlatformAzure() {
+        when(cloudPlatformConnectors.get(any(CloudPlatformVariant.class))).thenReturn(cloudConnector);
+        when(cloudConnector.encryptionResources()).thenReturn(encryptionResources);
+        EnvironmentDto environmentDto = EnvironmentDto.builder()
+                .withId(1L)
+                .withName("envName")
+                .withCloudPlatform("AZURE")
+                .withCredential(new Credential())
+                .withLocationDto(LocationDto.builder().withName("DummyRegion").build())
+                .withParameters(ParametersDto.builder()
+                        .withAzureParameters(AzureParametersDto.builder()
+                                .withEncryptionParameters(AzureResourceEncryptionParametersDto.builder()
+                                        .withEncryptionKeyUrl("dummyKeyUrl").build())
+                                .withResourceGroup(AzureResourceGroupDto.builder()
+                                        .withResourceGroupUsagePattern(ResourceGroupUsagePattern.USE_SINGLE)
+                                        .withName("dummyRG").build())
+                                .build())
+                        .build())
+                .withTags(new EnvironmentTags(new HashMap<>(), new HashMap<>()))
+                .build();
+        Environment env = new Environment();
+        env.setCloudPlatform("AZURE");
+        underTest.createEncryptionResources(environmentDto, env);
+        verify(encryptionResources, times(1)).createDiskEncryptionSet(any());
```

Comment:
Please add a stubbing (`when(..)`) logic for the `encryptionResources.createDiskEncryptionSet()` invocation and check if the response of `underTest.createEncryptionResources()` is as expected. That way there is no need for the `verify()` check.